deps/build.jl

```diff
@@ -121,7 +122,17 @@ end
 # need to be able to get the version before Python is initialized
 Py_GetVersion(libpy) = bytestring(ccall(Libdl.dlsym(libpy, :Py_GetVersion), Ptr{UInt8}, ()))
 
-const python = get(ENV, "PYTHON", "python")
+use_conda = false
+const python = try 
+    pyconfigvar(get(ENV, "PYTHON", "python"),"VERSION","")
```

Comment:
Probaby call `get(ENV, "PYTHON", "python")` only once, and check for Python >= 2.7.   i.e.
```jl
let py = get(ENV, "PYTHON", "python"), vers = VersionNumber(pyconfigvar(py,"VERSION","0.0"))
    if vers < v"2.7"
        error("Python version $vers < 2.7 is not supported")
    end
end
```